src/Microsoft.PowerShell.Commands.Diagnostics/ExportCounterCommand.cs

```diff
@@ -65,12 +65,13 @@ public string Path
                 ValueFromPipelineByPropertyName = false,
                 HelpMessageBaseName = "GetEventResources")]
         [ValidateNotNull]
+        [ValidateSet("blg", "csv", "tsv")]
```

Comment:
I've opened issue https://github.com/PowerShell/PowerShell-Docs/issues/1245 for this